tests/test_url_checks.py

```diff
@@ -1,58 +0,0 @@
-import pytest
```

Comment:
is this test being removed?

Reply:
After talking with the autopush folks, I only need what is in test_urls.py